sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
```

Comment:
I think, the documentation renderer is responsible for placing the section to the correct location. I used the original `KMeans` docstring as a reference, and it has `Notes` section above the `See also`. Maybe, we can keep it like this or change in both places.